front-end/src/components/Tables/SubmitTable.js

```diff
@@ -173,6 +173,7 @@ const SubmitTable = () => {
                     <MenuItem value={"women"}>Women</MenuItem>
                     <MenuItem value={"family"}>Family</MenuItem>
                     <MenuItem value={"other"}>Other</MenuItem>
+                    <MenuItem value={""}>Clear</MenuItem>
```

Comment:
See if you can get this option to show up _only_ if another option has already been selected. This might be helpful: [https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator)

Reply:
this has been fixed